include/AMOCommand.hpp

```diff
@@ -23,6 +26,16 @@ class AMOCommand {
   Action GetAction() const;
 
  private:
+  friend class boost::serialization::access;
+
+  template<class Archive> void serialize(Archive &ar,
+      const unsigned int version) {
```

Comment:
A const pass-by-value primitive doesn't make sense, not needed here and elsewhere.